src/keys-builder/utils/create-translation.ts

```diff
@@ -36,7 +45,7 @@ function createJson(config: CreateTranslationOptions) {
 }
 
 function createPot(config: CreateTranslationOptions) {
-  const resolved = resolveTranslation(config);
+  const resolved = flatten(resolveTranslation(config));
```

Comment:
Why do we always have to flatten the translation?

Reply:
If you prefer, this could be conditioned on `unflat = true`. Let me know.

Reply:
@rhuitl What was the motivation for always flattening it? 
If it's not a technical thing I think it should be conditioned to the config passed.

Reply:
> POT is flat, so there's no way a nested translation could be represented by it, no matter how the unflat option is set

In that case your code is ok 👍

Reply:
Already changed it, now it's consistent with the other unflatten usages. If you're happy with the current state, let's merge :)